CHANGELOG.md

```diff
@@ -31,6 +31,11 @@ version directory, and then changes are introduced.
 - Remove calico-ipip-pinger.
 - Remove calico-node-controller.
 
+## [v2.0.1]
+
+### Changed
+- Fix audit logging.
```

Comment:
The description you provide in this PR is way nicer than this here. I would like to see the more detailed version here. Fixing the audit log can mean anything.